include/aspect/stokes_matrix_free.h

```diff
@@ -118,6 +126,32 @@ namespace aspect
          */
         double pressure_scaling;
 
+        /**
+         * Table which stores the product of viscosity derivative with respect to pressure
+         * and newton derivative scaling factor.
+         */
+        const Table<2, VectorizedArray<number>>
+                                             *viscosity_derivative_wrt_pressure_table;
```

Comment:
I think it would look better if this is on one line or alligned with the beginning of the line